src/main/java/io/reactivesw/category/domain/service/CategoryService.java

```diff
@@ -165,14 +169,13 @@ public CategoryView getCategoryById(String id) {
    * @return the paged query result
    */
   public PagedQueryResult<CategoryView> queryCategories(QueryConditions queryConditions) {
-    LOG.debug("Enter queryCategories, QueryConditions is: {}.", queryConditions.toString());
-
+    LOG.debug("Enter. Query categories, query conditions is: {}.", queryConditions);
     List<Category> entities = categoryRepository.findAll();
 
     List<CategoryView> result = CategoryMapper.toModel(entities);
 
-    LOG.debug("End queryCategories, get Categories: {}.", result);
-
+    LOG.debug("Exit. Queried categories, the count of queried categories is: {}.", result.size());
+    LOG.trace("Exit. Queried categories, queried categories are: {}.", result);
```

Comment:
do'n need "Exit." here.
remove "Queried categories,".
"Cateogries: {}." is ok.